dxtbx/format/FormatHDFEigerNearlyNexus.py

```diff
@@ -156,7 +156,7 @@ def create_scalar(handle, path, dtype, value):
       handle['/entry/instrument/detector/x_pixel_size'].value)
     group['fast_pixel_direction'].attrs['transformation_type'] = 'translation'
     group['fast_pixel_direction'].attrs['vector'] = fast_axis
-    group['fast_pixel_direction'].attrs['offset'] = 0
+    group['fast_pixel_direction'].attrs['offset'] = (0, 0, 0)
```

Comment:
It's been a while since I looked at the Nexus stuff. Is offset really a vector rather than a scalar? Looking at http://download.nexusformat.org/sphinx/classes/applications/NXmx.html it isn't obvious to me.